src/grasshopper/lib/fixtures/__init__.py

```diff
@@ -380,25 +387,51 @@ def pytest_collect_file(parent, path):
 class YamlScenarioFile(pytest.File):
     """The logic behind what to do when a Yaml file is specified in pytest."""
 
+    composite_weighted_user_classes = {}
+
     def collect(self):
         """Collect the file, knowing the path via self.fspath."""
         # Third Party
         import yaml
 
-        raw = yaml.safe_load(self.fspath.open())
+        full_scenarios_list = yaml.safe_load(self.fspath.open())
         valid_scenarios = _get_tagged_scenarios(
-            raw_yaml_dict=raw, config=self.config, fspath=self.fspath
+            full_scenarios_list=full_scenarios_list,
+            config=self.config,
+            fspath=self.fspath,
         )  # tag filter
         for scenario_name, scenario_contents in valid_scenarios.items():
             test_file_name = scenario_contents.get("test_file_name")
+            child_scenarios = scenario_contents.get("child_scenarios")
             if test_file_name:
                 yield Scenario.from_parent(
                     self, name=scenario_name, spec=scenario_contents
                 )
+            elif child_scenarios:
+                import pathlib
```

Comment:
probably want to move this import to the top

Reply:
changed